pkg/apicapi/apicapi.go

```diff
@@ -193,7 +193,7 @@ func New(log *logrus.Logger, apic []string, user string,
 
 	conn := &ApicConnection{
 		ReconnectInterval: time.Second,
-		RefreshInterval:   30 * time.Second,
+		RefreshInterval:   time.Duration(refresh/2) * time.Second,
```

Comment:
Why not just do refresh * time.Second? We shouldn't have to do /2.

Reply:
Just to be pro-active and do the refresh before the APIC time's out.

Reply:
DONE